tests/ops/op_math/test_exp.py

```diff
@@ -536,17 +536,21 @@ def test_trotter_is_used_if_num_steps_is_defined(self):
         op = qml.IsingXY(phi, wires=[0, 1])
         exp = qml.evolve(op.generator(), coeff=-phi, num_steps=3)
         dec = exp.decomposition()
+        assert qml.math.allclose(
+            qml.matrix(qml.tape.QuantumScript(dec), wire_order=[0, 1]),
+            qml.matrix(exp, wire_order=[0, 1]),
+        )
         expected_decomp = [
-            qml.IsingXX(phi / 3, wires=[0, 1]),
-            qml.IsingYY(phi / 3, wires=[0, 1]),
-            qml.IsingXX(phi / 3, wires=[0, 1]),
-            qml.IsingYY(phi / 3, wires=[0, 1]),
-            qml.IsingXX(phi / 3, wires=[0, 1]),
-            qml.IsingYY(phi / 3, wires=[0, 1]),
+            qml.IsingXX(-phi / 3 / 2, wires=[0, 1]),
```

Comment:
Probably depends on the developer, but I would find it easier to read as (don't always have in mind the operation order in python)
```suggestion
            qml.IsingXX(-phi / (3 * 2), wires=[0, 1]),
```

Reply:
and same for the rest

Reply:
Could you help verify that the math is correct?

Reply:
Yes, its correct!